docs/swagger.yaml

```diff
@@ -1795,7 +1795,12 @@ paths:
                           type: boolean
                           description: Published or not
                           example: true
-                        
+                  metadata:        
+                    type: object
+                    description: metadata
+                    example:
```

Comment:
This has type `object` and should have a `properties` field as we have for the others.

Reply:
yes please